### PR TITLE
TSK-608: Move Vacancy support.

### PR DESCRIPTION
**Pull request by haiodo**
Signed-off-by: Andrey Sobolev <haiodo@gmail.com>

# Contribution checklist

## Brief description

* TSK-608
* Fix issue with live query updates.
* Fix issue with popup on wrong position

image

Screenshot 2023-02-07 at 16 10 50

## Checklist

* [x] - UI test added to added/changed functionality?
* [x] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [x] - Does Changelog.md is updated with changes?
* [x] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [x] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [x] - Is there any redundant or duplicate code?
* [x]  - Are required links are linked to PR?
* [x]  - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: plugins/view-resources/src/components/Table.svelte [hunk 1]
```diff
@@ -104,6 +104,7 @@
       query,
       (result) => {
         objects = result
+        console.log('OBJECTS', JSON.stringify(objects, undefined, 2))
```
Comment (BykhovDenis) [inline plugins/view-resources/src/components/Table.svelte]: Extra

**Review by SasLord — APPROVED** [review]

File: plugins/recruit-resources/src/components/MoveApplication.svelte [hunk 2]
```diff
@@ -0,0 +1,265 @@
+<!--
+// Copyright © 2020 Anticrm Platform Contributors.
```
Comment (BykhovDenis) [inline plugins/recruit-resources/src/components/MoveApplication.svelte]: We need way to manage copyrights)

**Review by BykhovDenis — APPROVED** [review]

File: plugins/view-resources/src/components/Table.svelte (hunk 1 above)
Comment (haiodo) [reply plugins/view-resources/src/components/Table.svelte]: fixed.

File: plugins/recruit-resources/src/components/MoveApplication.svelte (hunk 2 above)
Comment (haiodo) [reply plugins/recruit-resources/src/components/MoveApplication.svelte]: Fixed.

**Pull request merged**